birdhouse/config/jupyterhub/default.env

```diff
@@ -64,6 +64,15 @@ export JUPYTERHUB_CONFIG_OVERRIDE=""
 # recommended as it may permit unauthorized users from accessing jupyterhub.
 export JUPYTERHUB_AUTHENTICATOR_AUTHORIZATION_URL='http://twitcher:8000/ows/verify/jupyterhub'
 
+# 32 byte hex-encoded key used to encrypt a user's authentication state in the juptyerhub database.
+# If set, jupyterhub will periodically check if the user still has permission to access jupyterhub (according to Magpie)
+export JUPYTERHUB_CRYPT_KEY=
```

Comment:
What happens if the value is changed between restarts, causing a different state to be obtained?
Does Jupyter perform a new check with Magpie to verify if an updated value works, and update the state transparently with the new encryption key?
In other words, once this is set, must it remain the same, or we can automatically regenerate it each time?